internal/assets/assets.go

```diff
@@ -5,6 +5,11 @@ import (
 	iofs "io/fs"
 )
 
+const (
+	// PlaceholderFileName is the name of the file that is used to populate the directory structure of the embedded assets.
+	PlaceholderFileName = "placeholder"
```

Comment:
Is this to work around git not tracking empty folders? This might be more appropriate:

https://stackoverflow.com/a/932982/490321

Reply:
This is a workaround for assets. Without the placeholder files, it will not copy the directory structure.